include/aspect/particle/generator/interface.h

```diff
@@ -43,6 +43,7 @@ namespace aspect
       /**
        * Exception denoting a division by zero.
        */
+#if DEAL_II_VERSION_GTE(8,3,0)
       DeclExceptionMsg (ExcParticlePointNotInDomain,
                         "You requested to generate a particle at a position that "
```

Comment:
Ah, but then I also forgot that you can prepend everything with `template <int dim>` if you wanted.

But as I said, it's not important. We'll eventually get rid of the `#ifdef` again.